docs/source/storage/discovery_storage.rst

```diff
@@ -1,28 +1,49 @@
+ - **DESCRIPTION:** All users are given a ``/home`` directory automatically when their Discovery account is created. This storage is mainly intended for storing relatively small files such as script files, source code, and software installation files. While ``/home`` is permanent storage that is backed up and replicated, ``/home`` is not performant storage. ``/home`` also has a small quota, so you should frequently check your space usage (use a command such as, ``du -h /home/<yourusername>`` where ``<yourusername>`` is your user name, to see the total space usage). For running jobs and directing output files, you should use your ``/scratch`` directory.
  - **QUOTA:** 75GB
 
 **NAME:** ``/scratch/<yourusername>``
- - **DESCRIPTION:** You are given a ``/scratch`` directory automatically when your Discovery account is created. Scratch is a shared space for all users. The total storage available is 1.8PB; however, while this is performant storage, it is for temporary use only. **It is not backed up.** Data on ``/scratch`` should be moved as soon as possible to another location for permanent storage. You should run your jobs from ``/scratch`` and direct your output files to your ``/scratch`` directory for best performance, but it is best practice to move your files off of scratch to avoid any potential data loss.
+ - **DESCRIPTION:** All users are given a ``/scratch`` directory automatically when their Discovery account is created. Scratch is a shared space for all users. The total storage available is 1.8PB; however, while this is performant storage, it is for temporary use only. **It is not backed up.** Data on ``/scratch`` should be moved as soon as possible to another location for permanent storage. You should run your jobs from ``/scratch`` and direct your output files to your ``/scratch`` directory for best performance, but it is best practice to move your files off of scratch to avoid any potential data loss.
  - **QUOTA:** N/A
 
 **NAME:** ``/work/<groupname>``
- - **DESCRIPTION:** Research groups can request additional storage on ``/work``. A PI can request this extra storage through the `New Storage Space request <https://bit.ly/NURC-NewStorage>`_ . This is a performant, persistent, and long-term storage that is meant for storing data being actively used for research. It can be accessed by all members of the research group who have access permissions to this directory. 
+ - **DESCRIPTION:** Research groups can request additional storage on ``/work``. A PI can request this extra storage through the `New Storage Space request <https://bit.ly/NURC-NewStorage>`_. This is a performant, persistent, and long-term storage that is meant for storing data being actively used for research. ``/work`` can be accessed by all members of the research group who have access permissions to this directory. 
+- **QUOTA:** Each group can request up to **35TB** of free storage across all supplemental storage tiers: ``/work/<groupname>`` and ``/nese``.  
```

Comment:
This bullet isn't indented, but I wasn't sure how to fix it.

Reply:
Thank you for pointing that out. I corrected it and pushed the update. Sorry about missing it.